appwrite/website
# Add sql vs nosql blog post

## What does this PR do?

- Adds SQL vs NoSQL blog post and assets

## Test Plan

- Run the app and navigate to `/blog/post/sql-vs-nosql`
- Confirm that article and assets are displayed correctly
